bagua/torch_api/contrib/cache_dataset.py

```diff
@@ -0,0 +1,51 @@
+from torch.utils.data.dataset import Dataset
+from .cache_loader import CacheLoader
+
+__all__ = ["CacheDataset"]
+
+
+class CacheDataset(Dataset):
+    def __init__(
+        self,
+        dataset: Dataset,
+        backend: str = "redis",
+        key_prefix: str = "",
+        batch_writes: int = 20,
+        **kwargs,
+    ):
+        """
+        A dataset wrapper which caches `dataset` samples.
+
+        Args:
+            dataset: Dataset used for caching.
+            backend(str): The backend to use. Currently "redis" is supported.
+            key_prefix(str): Prefix of the cache key. Default ``""``.
```

Comment:
I cannot understand this. Documentation should be written in a way that new user can understand how to use this.

Reply:
Readers cannot understand what "cache key" is, and why they need "prefix" of that.